app/assets/javascripts/timelines_select_boxes.js

```diff
@@ -147,24 +147,22 @@ jQuery(document).ready(function($) {
     });
   });
 
-  $("#content").find("input").each(function (index, e) {
-    e = $(e);
-    if (
-        ((e.attr("type") === "text" || e.attr("type") === "hidden") && e.val() !== "" && !e.hasClass("select2-input")) ||
-        (e.attr("type") === "checkbox" && e.attr("checked"))
-    ) {
-      showFieldSet(e);
-    }
-  });
+  var fields = $("#content").find("input")
+                            .not("[type='radio']")
+                            .not("[class^='select2-']")
+                            .not(".button")
+                            .not("[type='hidden']")
```

Comment:
@myabc It's fine, it's fine, i am just not used to writing pure JavaScript anymore.